src/app/public/src/modules/image/image.component.spec.ts

```diff
@@ -34,4 +34,10 @@ describe('StacheImageComponent', () => {
     expect(component.imageSource).toBe(imageSource);
   });
 
+  it('should have a description input', () => {
+    component.description = this.description;
```

Comment:
We should define `description` for readabilities sake

Reply:
Also a test to be sure the value is undefined should the description not be set by a user would be valued as it is conditional.